contributing.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to AcadeMe
+
+Thank you for supporting AcadeMe and looking for ways to help. 
+We surely apperciate open source contributers! But before you contribute, please make sure to follow these guidelines.
+
+## Setting up
+1. Fork the repository! <img align="right" width="250" src="https://camo.githubusercontent.com/fcf9a4ed664cc63de2fcb14d1135072ba6d4c74a8e9bdb224ad6ab1e72600c3b/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f666f726b2e706e67">
+2. Clone the repository!
+
+## Issues
+
+### Bug Reports
+
+We use GitHub [issues](https://github.com/redhat-beyond/AcadeMe/issues) to track public bugs. Report a bug by opening a new issue; it's that easy!
+Great Bug Reports tend to have:
+
+- A quick summary and/or background.
+- Steps to reproduce.
+- Be specific!
+- Give sample code if you can.
+- What actually happens.
+- Make it visual - use photos and screenshots if you can.
+
+People love thorough bug reports. I'm not even kidding.
+
+
+## Pull requests
+
+<img align="right" width="250" height="150" src="https://theindecisiveeejit.files.wordpress.com/2014/12/size-matters.jpg">
+
+- Make SMALLER pull requests. 
+- Write useful descriptions and titles. 
+    Example: "[Frontend] Login Page UI" 
+             "I have made several changes to the Login Page interface: 1.'forgot password' checkbox 2.Logo alignment"
+- Have on-point commit messages.
+
```

Comment:
add something about linking the issue (bug report) to the pr, in the first pull request message body.

Reply:
Oh, and link an issue to this very PR. :)